src/components/NavigatorSearch/useQuickSearch.ts

```diff
@@ -28,7 +28,7 @@ function useQuickSearch(open: boolean, input: RefObject<HTMLInputElement>): UseQ
   const { result: customers, search: searchCustomers } = useCustomerSearch({
     variables: {
       ...DEFAULT_INITIAL_SEARCH_DATA,
-      first: 5,
+      first: 10,
```

Comment:
question: is there any reason to change this to 10?

Reply:
I thought that 5 results were too little for a search like that.